app/models/gpttags.server.test.ts

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import { requestGpttags } from "../code/scout/gpttags";
+
+// Mocking axios.post method
+jest.mock("axios");
+
+describe("requestGpttags", () => {
```

Comment:
in general we would want a gptApi module that let's us pass in a prompt (string) and get a response(string).  this module would handle all the details like deciding the GPT model and pulling the "relevant response" from the json blob that GPT sends back.

Mocking THAT would be much simpler for a test.

Reply:
ah yes, and you created such a thing:
requestGpttags

now that function name is not accurate to what the function does, but it's a great function to have.  Let's mock THAT instead of lower level GPT fetch request.